README.md

```diff
@@ -34,19 +32,43 @@ repo forall -p -c 'git checkout $REPO_RREV'
 ```
 
 ## Build Instructions
+
 Each component can be built with CMAKE, or use the [tools/maptools.py](tools/README.md) build command.
 
-To build everything the first time, with all features enabled, run
-```
+To build prplMesh in debug mode (for being able to debug with gdb), with all features and tests, run
+
+```bash
 cd prplMesh/tools
-./maptools.py build all -n -f MSGLIB=zmq BWL_TYPE=DWPAL BUILD_TESTS=ON
+./maptools.py build map -n -f MSGLIB=zmq BUILD_TESTS=ON CMAKE_BUILD_TYPE=Debug
 ```
 
-Subsequent builds don't need to repeat all of these options. Also, you don't
-need to re-build the dependencies when you're developing prplMesh itself.
-```
+Subsequent builds don't need to repeat all of these options:
+
+```bash
 ./maptools.py build map
 ```
 
 Run `./maptools.py build --help` for more options.
 
+Note - to build with Docker, see provided [building with docker](tools/docker/builder/README.md)
+
+## Running Instructions
+
+Once built, prplMesh controller, agent and framework can be started using `prplmesh_utils.sh`:
+
+```bash
+cd ~/prplMesh/build/install/scripts
```

Comment:
Above, we assume the current directory is just above prplMesh (i.e. the root of the manifest checkout), so the path should be relative to that. We definitely can't assume that it's checked out in `~/prplMesh`.
```suggestion
cd build/install/scripts
```